### PR TITLE
Properties, adds write support and reads list from task instead of model

### PR DESCRIPTION
**Pull request by planthaber**

### CODE REVIEW

**Comment by planthaber** [comment]
Added property writing ability

File: lib/rock/webapp/tasks/api.rb [hunk 1]
```diff
@@ -101,28 +101,48 @@ def get_operation(name_service, name, operation_name)
...
                     end
                     
+                    desc "writes a property"
+                    post ':name_service/:name/properties/:property_name/write' do
+                        task = task_by_name(params[:name_service], params[:name])
+                        prop = task.property(params[:property_name])
+                            
+                        begin
+                            obj = MultiJson.load(request.params["value"])
+                            puts obj.pretty_inspect
```
Comment (doudou) [inline lib/rock/webapp/tasks/api.rb]: .... remove

**Comment by doudou** [comment]
Please add tests.

File: lib/rock/webapp/tasks/api.rb (hunk 1 above)
Comment (planthaber) [inline lib/rock/webapp/tasks/api.rb]: I keep forgetting removing them, sorry

**Comment by planthaber** [comment]
done, the other tests are failing with corba exceptions though...

File: test/test_tasks.rb [hunk 2]
```diff
@@ -75,6 +77,39 @@ def with_stub_task_context(task_name)
...
+                with_stub_task_context "task" do |task|
+                    get "/tasks/localhost/task/properties"
+                    assert_equal 201, last_response.status
+                    model = task.model.properties.to_h
+                    properties = Array.new
+                    expected = Hash[properties: properties]
+                    assert_equal expected, MultiJson.load(last_response.body, symbolize_keys: true)
+                end
+            end
+
```
Comment (doudou) [inline test/test_tasks.rb:92]: What about a property read test ?

Comment (planthaber) [inline test/test_tasks.rb:92]: Oh, read was available before, expected that is was already there, just added it

**Pull request merged**